DBM-GUI/DBM-GUI.lua

```diff
@@ -518,7 +513,7 @@ function DBM_GUI:CreateBossModPanel(mod)
 					else
 						local _title = DBM:GetSpellInfo(spellID)
 						if _title then
-							title, desc, icon = _title, tonumber(spellID), GetSpellTexture(spellID)
+							title, desc, icon = _title, tonumber(spellID), GetSpellTexture(spellID or 0)
```

Comment:
What's going on with this line?

Reply:
LuaLS isn't smart enough to deduce that spellID is guaranteed non-nil here, so the "or 0" is the quickest way to make it happy